clients/elrond/client.go

```diff
@@ -39,6 +41,7 @@ type ClientArgs struct {
 	IntervalToResendTxsInSeconds uint64
 	TokensMapper                 TokensMapper
 	MaxRetriesOnQuorumReached    uint64
+	ProxyCacherExpirationTime    uint64
```

Comment:
Since we multiply this with time.Second, can we rename this to `ProxyCacherExpirationSeconds` ?